src/at_tools/att_relevant_files.rs

```diff
@@ -293,11 +484,63 @@ const RF_WRAP_UP_DEPTH: usize = 5;
 const RF_WRAP_UP_TOKENS_CNT: usize = 8000;
 
 
+const SUPERCAT_DECIDER_PROMPT: &str = r###"
+Read slowly and carefully the problem text one more time before you start:
+
+{USER_QUERY}
+
+In the previous message you were given a generous context -- skeletonized files.
+
+TODO:
+1. analyse thoroughly the problem statement;
+2. analyse thoroughly context given (skeletonized files from the previous message);
+3. among the files pick the one you need to make changes to solve the problem (according to the problem statement);
+4. among the files pick at least 5 more files that will give you the best context to make make changes in the chosen file (from step 3);
+5. return the results in a format specified below;
+
+Format thee must obey:
```

Comment:
thee?

Reply:
2nd form, informal

Reply:
2nd form of what?

Reply:
in English there is "you" -- 2nd form formal and "thou" -- 2nd form informal. Thou -- thee -- thy -- thine

Reply:
So why do you use it instead of 'you'?

Reply:
Because I treat it as "ты", not as "вы"

Reply:
> Thee is an old-fashioned, poetic, or religious word for `you' when you are talking to only one person


You seriously need this here?

Reply:
Thou thinkest it might affect how well the model understands prompt?

Reply:
I don't understand it and some other people as well might not know this word. Don't make it more difficult than it is 

Also it might be out-of-domain for some smaller models

Reply:
Ok 👌🏻